Statistics/Distributions/Uniform.cs

```diff
@@ -38,17 +39,17 @@ public class Uniform: IDistribution, IValidate<Uniform>
         public Uniform()
         {
             //for reflection
-            _Distribution = new MathNet.Numerics.Distributions.ContinuousUniform(0, 1);
+            Min = 0;
```

Comment:
I roughly understand what microsoft has written about reflection. Could you give me an example of when/why you would use reflection and this constructor?

Reply:
it is because i use activator create instance of when i read in a distribution from xml. I use the default constructor, and i set the appropriate default values so that i can load in the values from xml and then call the appropriate method to build from properties.

since i cannot enforce a constructor signature through an interface, that means i need to rely on a default constructor with no arguments for deserialization